.travis.yml

```diff
@@ -2,11 +2,11 @@ language: python
 python: 3.6
 cache: pip
 
-
-before_install:
-- pip install -U pre-commit
-- pip install django==1.11
-
 script:
-- pre-commit run --all-files
 - python ./manage.py test
+
+jobs:
+  include:
+    - stage: pre-commit tests
```

Comment:
@PetrushynskyiOleksii this is also a sequence item to dedent

Reply:
ok, i'll correct it